frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/DistributionField.tsx

```diff
@@ -3,7 +3,7 @@
 import { AcmButton, AcmInlineStatus, StatusType } from '@open-cluster-management/ui-components'
 import { ButtonVariant } from '@patternfly/react-core'
 import { ArrowCircleUpIcon, ExternalLinkAltIcon } from '@patternfly/react-icons'
-import { ReactNode, useState } from 'react'
+import React, { ReactNode, useState } from 'react'
```

Comment:
This looks weird to me...

Reply:
Thanks for pointing that out, was totally unnecessary import.